scale-ui/app/modules/jobs/partials/jobDetailTemplate.html

```diff
@@ -27,11 +27,20 @@ <h4 class="panel-title error-title"><strong>{{ vm.job.error.category }} Error: {
                         {{ vm.job.error.description }}
                     </div>
                 </div>
+                <div class="panel panel-info" ng-show="vm.job.is_superseded">
+                    <div class="panel-heading">
+                        <div class="panel-title">
+                            <strong>SUPERSEDED {{ vm.supersededChange }}</strong>
+                        </div>
+                    </div>
+                </div>
```

Comment:
I did see that comment and it behaves just as you suggested. 
![image](https://cloud.githubusercontent.com/assets/15909313/18966709/444739fe-8650-11e6-86ba-8bb63918ac8a.png)

As far as colors, I think we need to rethink the way we display statuses. It's hard to understand the colors unless you're working with them all day every day.